src/modules/auth/service.js

```diff
@@ -1,11 +1,62 @@
 const jwt = require('jsonwebtoken');
 const { AUTH } = require('../../config');
+const { UnauthorizedError } = require('../../utils/errors');
+const { INVALID_TOKEN_MESSAGE } = require('../auth/constants');
+const userService = require('../user/service');
 
 const generateAccessToken = (
   payload,
-  options = { expiresIn: AUTH.TOKEN_EXPIRATION_TIME },
-) => jwt.sign(payload, AUTH.SECRET, options);
+  options = { expiresIn: AUTH.ACCESS_TOKEN_EXPIRATION_TIME },
+) => jwt.sign(payload, AUTH.ACCESS_TOKEN_SECRET, options);
+
+const generateRefreshToken = (
```

Comment:
You dont need an extra method for generate refresh token
You can reuse generateAccessToken method
But please rename it in more common way